test/test_read_preferences.py

```diff
@@ -265,7 +265,7 @@ def test_nearest(self):
 
         not_used = data_members.difference(used)
         latencies = ", ".join(
-            "%s: %dms" % (server.description.address, server.description.round_trip_time)
+            "%s: %dms" % (server.description.address, server.description.round_trip_time)  # type: ignore[str-format]
```

Comment:
Is it possible to fix the format string to remove the type ignore?

Reply:
I believe the error is that round_trip_time is an `Optional[float]`, my first instinct was to add an `assert server.description.round_trip_time is not None` statement but that doesn't work well with the list comprehension...alternatively, I could cast it? What are your thoughts?

Reply:
Can we use `%s` instead of `%d`?

Reply:
yes! good idea!